libs/blocks/table/table.js

```diff
@@ -153,6 +153,58 @@ function handleAddOnContent(table) {
   table.addEventListener('mas:resolved', debounce(() => { handleEqualHeight(table, '.row-heading'); }));
 }
 
+function setTooltipPosition(el) {
+  const tooltips = el.querySelectorAll('.milo-tooltip');
+
+  tooltips.forEach((tooltip) => {
+    const classesToCheck = ['top', 'bottom', 'left', 'right'];
+    const defaultClass = classesToCheck.find((className) => tooltip.classList.contains(className)) || 'right';
+    let isTabletOrMobile = ['TABLET', 'MOBILE'].includes(defineDeviceByScreenSize());
```

Comment:
I have a suggestion here to avoid updating the value in multiple places.
```js
const isTabletOrMobile = () => ['TABLET', 'MOBILE'].includes(defineDeviceByScreenSize());
```
Then you can use it everywhere as `isTabletOrMobile()`